src/models/Card/Card.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export interface Card {
+    cardName: string;
+    cardNumber: string;
+    cardType: string;
+    companyId: string;
+    creditLimit: null;
+    expensesTypes: any;
```

Comment:
Need to strong type this. I'm assuming `espensesTypes` is an array of objects? Perhaps, create an `ExpenseType` object and set this property to be of `ExpenseType[]` type.

### _Remark:_ 
I am skeptical of the plurality of this property name. Is this meant to be array of objects or just 1 singular object? The `expenseTypes` in the `UpdateCardForm` interface defined in `src/clients/FlexbaseClient.Card.ts` was incorrectly named to be plural when it only represents a singular object. 

Is the `espensesTypes` defined here supposed to be different to the aforementioned `expenseTypes` in the `UpdateCardForm` interface?

Reply:
`expensesTypes` was the name that the API assign to this object. I just put the same name as the API response in the Interfaces